src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        String color = String.valueOf(colorSupplier.getRandomColor());

```

Comment:
Use `name()` for getting `String` representation of enum constants instead of `String.valueOf()`. This ensures that even if `toString()` is overridden in the enum, you will get the correct name of the enum constant.